containers/init/build/php/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM php:7.2.22-buster
+RUN mkdir -p /src/workspace
+WORKDIR /src/workspace
+
+RUN apt-get update && apt-get install -y \
+  git \
+  zlib1g-dev \
+  build-essential \
+  lcov \
+  rubygems \
```

Comment:
In my understanding, rubygems should be installed alongside ruby when the installation works properly. This is the case for rbenv at least. I believe this was also the behavior for rvm, but it has been years since I have used it.

Reply:
I have tried without installing rubygems, it just doesn't work...

Reply:
I think I have tried without it and doesn't work, but I think you could be right, I have tried too many combinations and lost track of what I have altered, let me try again and fix it.

Reply:
You are right, we do not need rubygems installed , fixed